iosTraining-Reeen/WeatherViewController.swift

```diff
@@ -62,24 +62,44 @@ class ViewController: UIViewController {
         return stackView
     }()
     
+    private var weatherService: WeatherServiceProtocol
+    
+    init(weatherService: WeatherServiceProtocol) {
+        self.weatherService = weatherService
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         setupViews()
     }
 }
 
-private extension ViewController {
+private extension WeatherViewController {
+    func displayWeatherCondition() {
+        let conditionString = self.weatherService.getWeatherInformation()
+        weatherConditionImageView.image = imageForWeatherCondition(conditionString)
+    }
+    
     func setupViews() {
         view.backgroundColor = .white
         view.addSubview(weatherConditionStackView)
         view.addSubview(closeButton)
         view.addSubview(reloadButton)
         
-        weatherConditionStackView.snp.makeConstraints { make in
+        reloadButton.addAction(UIAction(handler: { [weak self] _ in
+            self?.displayWeatherCondition()
+        }), for: .touchUpInside)
+        
+        weatherConditionImageView.snp.makeConstraints { make in
```

Comment:
コンフリクト解消時にミスってるかもです。
```suggestion
        weatherConditionStackView.snp.makeConstraints { make in
```

Reply:
StackViewの制約が消えていました。